hnn_core/tests/test_params.py

```diff
@@ -13,3 +14,33 @@ def test_params():
     params = Params(params_fname)
     print(params)
     print(params['L2Pyr*'])
+
+
+def test_legacy_params():
+    """Test legacy params object."""
+    hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
+    param_url = ('https://raw.githubusercontent.com/hnnsolver/'
+                 'hnn-core/test_data/default.param')
+    params_legacy_fname = op.join(hnn_core_root, 'param', 'default.param')
+    if not op.exists(params_legacy_fname):
+        _fetch_file(param_url, params_legacy_fname)
+
+    params_new_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params_legacy = Params(params_legacy_fname)
+    params_new = Params(params_new_fname)
+
+    if not params_new == params_legacy:
+        for key in params_new.keys():
+            if key not in params_legacy:
+                print("no value for %s in %s" %
+                      (key, op.normpath(params_legacy_fname)))
```

Comment:
normally we test using `pytest`. So you should use `assert` statements instead of `print`. So you stop at the first failure